docs/sphinx/ome-xml/index.rst

```diff
@@ -1,25 +1,13 @@
 The OME-XML format
 ==================
 
-OME-XML is a project to provide support for the exchange of microscope
-imaging data within the field of life sciences. Its key aspects include
-support for collaborative sharing of data between labs who may be using
-different apparatus, and the safeguarding of crucial metadata associated
-with image files, that can be lost when they are moved across systems (or
-in many cases, never recorded at all).
+OME-XML is a `file format <https://en.wikipedia.org/wiki/File_format>`_
+for storing microscopy information (both pixels and metadata) using the
+OME-XML data model.
```

Comment:
Is this meant to be "OME data model" as per the PR description?

Reply:
Well spotted, it should be for consistency, yes

Reply:
Yes, updated now.